### PR TITLE
CIS-277 Update examples for unread count

### PR DESCRIPTION
**Pull request by buh**
- Fixed CHANGELOG for #369 
- Updated examples and docs for unread count usage

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,14 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Upcoming
 
-### ✅ Added
-- Pipelining is enabled for requests to improve them between clients and regions over long distances. 
+### 🐞 Fixed
+- Pipelining is enabled for requests to improve them between clients and regions over long distances  [#396](https://github.com/GetStream/stream-chat-swift/pull/396)
```
Comment (VojtaStavik) [inline CHANGELOG.md:10]: I'll try again. Let's improve the wording a little bit:
`HTTP pipelining is enabled to improve request response times for situations when the clients and servers are in different regions.`

File: Example/Sources/RootViewController.swift [hunk 2]
```diff
@@ -124,7 +124,7 @@ final class RootViewController: ViewController {
             guard let self = self else { return }
             
             // Request 100 messages to get the initial value of unread count.
```
Comment (VojtaStavik) [inline Example/Sources/RootViewController.swift:126]: The comment should be updated too?

**Comment by VojtaStavik** [comment]
@buh This can be merged, too, right?

**Comment by buh** [comment]
Yes. I’ll fix conflicts

**Pull request merged**